Further tidy on Android instructions README.md

```diff
@@ -949,24 +949,24 @@ mv model.gguf ~/
 [Get the code](https://github.com/ggerganov/llama.cpp#get-the-code) & [follow the Linux build instructions](https://github.com/ggerganov/llama.cpp#build) to build `llama.cpp`.
 
 #### Building the Project using Android NDK
-You can easily run `llama.cpp` on Android device with [termux](https://github.com/termux/termux-app#installation).
-
-First, install the essential packages for termux:
-```
-pkg install clang wget git cmake
-```
-Second, obtain the [Android NDK](https://developer.android.com/ndk) and then build with CMake:
-
-You can execute the following commands on your computer to avoid downloading the NDK to your mobile. Of course, you can also do this in Termux.
+Obtain the [Android NDK](https://developer.android.com/ndk) and then build with CMake:
```

Comment:
NDK install is usually done on pc, so referring to termux doesn't make sense at this point.